aws/resource_aws_emr_cluster_test.go

```diff
@@ -98,6 +98,17 @@ func TestAccAWSEMRCluster_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "bootstrap_action.#", "0"),
 				),
 			},
+			{
```

Comment:
This should have an Import test after the first step, as well as after, to ensure that `terraform import` will work in both cases